src/shared/components/mutationTable/column/CategoricalColumnFormatter.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+
+export interface ICategoricalColumn {
+    displayValue?: string;
+    toolTip?: string;
+    className: string;
+}
+
+export default class CategoricalColumnFormatter {
+    /**
+     * Boilerplate code to create tool tip.
+     */
+    public static createToolTip(content: JSX.Element, toolTip: string) {
+        const arrowContent = <div className="rc-tooltip-arrow-inner" />;

```

Comment:
i think this is actually the default arrowContent for DefaultToolTip, so maybe no necessary

Reply:
yes, you are right, that this is default `arrowContent`, and this code is not necessary.  Have now removed.